tests/test_background_tasks.py

```
@@ -805,3 +829,895 @@ def mock_list_objects_v2(**kwargs):
+
+    with session_scope(engine) as session:
+        refreshed = session.get(User, user_id)
+        assert refreshed is not None
+        assert refreshed.storage_used == 5000  # 8000 - 3000
+
+    # Cleanup
+    with session_scope(engine) as session:
+        session.query(User).filter(User.id == user_id).delete()
+
+
+def test_process_single_photo_skips_when_photo_not_in_existing_ids(engine: Engine, s3_container, monkeypatch) -> None:
```

Ruff (0.15.20)

[warning] Unused function argument: `s3_container`

(ARG001)